src/views/usages/UsagesTree/CodeUsageNode.ts

```diff
@@ -117,6 +118,15 @@ export class CodeUsageNode extends vscode.TreeItem {
       children.push(variableDetailsRoot)
     }
 
+    if (!('variable' in match)) {
+      const notFoundNode = new CodeUsageNode(
+        key + ':not-found',
+        '',
+        'detail'
+      )
+      notFoundNode.description = 'VARIABLE NOT FOUND IN DEVCYCLE'
```

Comment:
Is there any particular reason this field isn't set through the class constructor? I dont think it makes a big difference but Im just curious

Reply:
oh i just didn't want to pass an empty array for the children parameter. Thought this might look cleaner. Okay with changing it - like you said, it doesn't make a big difference

Reply:
nah its fine, was just wondering